README.md

```diff
@@ -72,6 +72,11 @@ except ApiException as e:
 # except ApiException as e:
 #     print("Exception when calling add_to_blacklist: %s\n" % e)
 
+# try:
+#     api_instance.is_blacklisted('+37255555555')
+# except ApiException as e:
+#     print("Exception when calling is_blacklisted: %s\n" % e)
```

Comment:
does this in general mean that checking for number in blacklist will result in ApiException if the number is not blacklisted?

Reply:
If the number is not blacklisted, then you get
```
Exception when calling is_blacklisted: (404)
Reason: NOT FOUND
HTTP response headers: HTTPHeaderDict({'Date': 'Tue, 08 Jan 2019 14:31:19 GMT', 'Content-Type': 'application/json', 'Content-Length': '114', 'Connection': 'keep-alive'})
HTTP response body: {"errors":[{"code":"203","detail":"the phone number was missing from the blacklist","title":"Missing resource"}]}
```

Reply:
Yeah, so perhaps it would be better if in the examples the more specific Exception would be caught/handled first. Otherwise it's quite difficult to understand that this actually not an error case.

Reply:
The example provides extra info. That's enough.

Reply:
The error has extra info. That's enough.

Reply:
Does not seem very developer friendly. How do I programmatically tell the difference between a 503 and 404 error? 
The example does not provide a way to do this (e.g how do I actually access the body/HTTP status code in case of an exception?

Reply:
Out of the scope of the example.

Reply:
I agree with both of you - first, it is not developer friendly and hard to differentiate between 503 and 404 error, however this is more general problem with the library and should be solved in a separate PR.